tests/vcs/git/GitCommitBearTest.py

```diff
@@ -100,6 +100,24 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_github_pull_request_temporary_merge_commit(self):
+        self.run_git_command('remote', 'add', 'upstream',
+                             'git://github.com/kriti21/coala-bears.git')
```

Comment:
> Then if the tester is offline, they will at least see that one of the tests is successful, and the test which requires GitHub will fail due to no internet connection.

We can create a GitHub PR merge commit by pulling a PR only which requires internet connection. Otherwise these tests fail already due to giving timeout error in the fetch statement only. There is no other way to create this magic merge commit offline.

Reply:
@kriti21 If GitHub can create it, then we can create it as well. It's basically a merge commit that's created by using [git merge](https://git-scm.com/docs/git-merge)